mappings/net/minecraft/test/TestBatchUtil.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS net/minecraft/unmapped/C_doimveny net/minecraft/test/TestBatchUtil
```

Comment:
really minor change, but i think `TestBatchingUtil` has a better sound. up to you!